articles/explainable-ai/index.md

```diff
@@ -0,0 +1,160 @@
+
+#### Human Factor and AI Frameworks Exhibiting Bias
+A huge selling point of machine learning algorithms is that they aim to remove human intervention from various processes. We trust that these AI systems are fair, accurate, and objective. However, what we fail to consider is that they are created human beings. Human beings possess various biases.
+
+It is very easy for creators of machine learning models to impart such biases into the models. As mentioned above, COMPAS was found to be guilty of [racial bias](https://www.propublica.org/article/machine-bias-risk-assessments-in-criminal-sentencing). It was meant to determine which inmates were least likely to recommit offenses.
+
+However, it ended up inaccurately flagging black inmates as a higher risk to re-offend when compared to white inmates. If an algorithm is to determine your freedom, it is only fair that you get to know how the decisions were made.
+
+Another case of bias involved a digital application for a mortgage. An algorithm was said to charge credit-worthy minorities [higher interest](https://www.americanbanker.com/news/werent-algorithms-supposed-to-make-digital-mortgages-colorblind) rates than white applicants. Explainable AI can help reduce these incidents of bias.
+
+If similar algorithms offered clear explanations on why the rates were decided upon, some of the consequences of bias would be contained before wreaking widespread harm. The bias would be easier to pick up. The algorithms would be fine-tuned to provide better and fairer experiences to all users.
+
+I recently wrote an article on bias in machine learning and how to handle it. To get a better understanding on bias, check it out [here](/engineering-education/handling-bias-in-machine-learning/).
+
+#### Models are More Complex and Unpredictable
+AI methods are usually either simple or complex. Simple methods are often rule-based and explainable. For example, a [decision tree](https://towardsdatascience.com/decision-trees-in-machine-learning-641b9c4e8052) model. This is a tree-like graph that splits a data set based on differing conditions. It is popularly used in classification problems, though it is also used for regression tasks. This type of model is very easy to understand and visualize for the user.
+
+On the other hand, complex methods are designed without explainability in mind. These methods often provide greater performance than simple AI methods. They are not designed to be interpretable. The advancements of deep learning, coupled with an increase in computational power (accessible for the algorithms), makes these complex methods even more unpredictable. Explanations become foggier. A trade-off between explainability and performance emerges.
```

Comment:
I think this paragraph is key in understanding the topic of the article. 
I had to read it a couple times to get what it was trying to convey. 
Perhaps going into some details as to how and why they are not interpretable ( perhaps detailing the added layers of complexity) in using neural networks or something around those line - to make it clear to the reader as to why when it comes to more complex methods we lose the explainability.

Reply:
Hey @hectorkambow, noted. Made some changes, let me know if they are satisfactory